frontend/app/views/offer/subscriber.scala.html

```diff
@@ -0,0 +1,93 @@
+@import model.Benefits
+@import views.support.Prices._
+
+@main("Subscribers") {
+    @defining(Benefits.details(Tier.Partner)) { benefits =>
+        <main role="main">
+
+            <div class="page-slice page-slice--slim l-constrained l-side-margins">
+            @fragments.offer.callout("images/offer/subscriptions/audience-1140.jpg", "Pay nothing for 6 months")
+            </div>
+
+            <section class="tone-white l-gutter l-constrained">
+                <div class="page-section page-section--slim">
+                    <div class="page-section__content">
+                        <p class="text-feature">
+                            A unique opportunity for subscribers to experience the partner tier of Guardian membership (worth £15/ month) free for 6 months.
+                        </p>
+                        <a class="action u-no-margin" href="@routes.Joiner.tierList">Join now</a>
+                    </div>
+                </div>
+            </section>
+
+            <section class="tone-white tone-featured-tint l-gutter l-constrained">
+                <div class="page-section page-section--slim">
+                    <div class="page-section__content">
+                        <h2 class="h-feature">What's included</h2>
+                        <ul class="package-benefits grid grid--3up">
+                        @for(benefit <- benefits.list){
+                            <li class="grid__item">
+                                <div class="u-divider-dotted">@fragments.benefits.item(benefit)</div>
+                            </li>
+                        }
+                        </ul>
+                    </div>
+                    <div class="page-section__supplementary">
+                        <aside class="aside-feature" role="complementary">
+                            <p class="text-feature">
+                                Get closer to the stories that matter and experience the Guardian brought to life, with priority booking and discounted tickets</p>
+                            <div>
+                                <a class="action" href="@routes.Joiner.tierList">Join now</a>
+                            </div>
+                            <div>
+                                <a class="action action--secondary u-no-margin" href="@routes.Info.about">
+                                    More about Membership</a>
+                            </div>
+                        </aside>
+                    </div>
+                </div>
+            </section>
+
+            <section class="tone-white l-gutter l-constrained">
+                <div class="page-section page-section--slim">
+                    <div class="page-section__content">
+                        <h2 class="h-feature">Subscriber offer</h2>
+                        @for(pricing <- benefits.pricing) {
+                            @fragments.offer.option(
+                                title="Pay annually",
```

Comment:
Rather than having the `rightGutter` option, is there an oportunity to to use the grid component here?